website/src/parsers/java/java-parser.js

```diff
@@ -0,0 +1,51 @@
+import defaultParserInterface from '../utils/defaultParserInterface';
+import pkg from 'java-parser/package.json';
+
+const ID = 'java-parser';
+
+export const parserSettingsConfiguration = {
+  fields: [],
+};
+
+export default {
+  ...defaultParserInterface,
+
+  id: ID,
+  displayName: ID,
+  version: pkg.version,
+  homepage:
+    pkg.homepage ||
+    'https://github.com/jhipster/prettier-java/tree/master/packages/java-parser',
+
+  locationProps: new Set(['location']),
+  typeProps: new Set(['name']),
+
+  loadParser(callback) {
+    require(['java-parser'], callback);
+  },
+
+  parse(parser, code) {
+    const cst = parser.parse(code);
+
+    const clone = JSON.parse(JSON.stringify(cst));
```

Comment:
Thank you for doing this! I still don't know why this line is necessary.... any ideas?

Reply:
This was possible debug code by the author. I've removed it.